### ISSUE TITLE
change ordering of observations 

### ISSUE THREAD

**Issue by jrhillae** [inbo/vespa-watch#427]
Dominique Soete, who uses the website very intensively, says that it makes more sense to order observations of Vespa Velutina (https://vespawatch.be/obs/) based on the date of notification instead of the date of observation. Often people only put the observation on the website a few days later and then it is never visible on this webpage as a new observation (https://vespawatch.be/obs/). As such, it takes more time for him to trace new observations. I suppose this is only a matter of reordering observations based on the parameter 'created at', instead of 'observation time'?

Thank you.

**Comment by niconoe** [comment]
Indeed, it appears the change is technically simple, as you suggested (2 tiny changes in the `get_observations()` method, if you want to have a try).

I do however a few questions related to the clarity of the user interface:

- Should we implement this change on both the index page (4 observations at the bottom) and the larger list (https://vespawatch.be/obs/), or only on the latter?
- Should we also change the displayed date (under the picture) to the date of `created_at`? Not doing so could give the impression to a visitor that the observations are unordered. Or maybe we could solve that by showing *both* the dates there?
- Please also note that `created_at` will always reflect the time the observation is inserted in the database behind the Django app. So if an observation is submitted via VespaWatch, that will be the "correct", but if an observation is initially submitted on iNaturalist then pulled on VespaWatch, it will reflect the datetime where the  the synchronisation process occurred (usually twice per day, but that can rarely be delayed for a few days if a synchronisation issue occurs and has to be fixed manually)

Can I have your thoughts about all this? Thanks

**Comment by jrhillae** [comment]
-If it possible it would be handy to have the change implemented at both the index page and the larger list (https://vespawatch.be/obs/)

-Maybe we can show both dates for clarity? That seems the best suggestion.

**Comment by jrhillae** [comment]
I tried to add both the observation date and date of creation to an observation card. Does this look ok?

![Screenshot from 2021-08-17 15-49-28](https://user-images.githubusercontent.com/35692395/129737870-f80619d8-b196-4967-9def-e2d8e3e317cb.png)

**Comment by niconoe** [comment]
That looks good, @jrhillae, congratulations!

If you want to submit a pull request for this change, I can review it and then deploy that to the test (UAT) website, so we can have a last check before sending it to the production website!

Thanks again for your contribution
